poolapi/apijson.go

```diff
@@ -0,0 +1,43 @@
+package poolapi
+
+import (
+	"encoding/json"
+)
+
+type Response struct {
```

Comment:
Maybe this needs a "type" field so the client can double check the `Data` field is what they expect based on the request?